ai-service/tests/conftest.py

```
@@ -0,0 +1,33 @@
+import pytest
+import chromadb
+from dotenv import load_dotenv
+import os
+from ai_service.exceptions import NotFound
+from ai_service import db
+
+
+def create_db_test_collection(collection_name: str):
```

Ruff (0.12.2)

Missing return type annotation for public function `create_db_test_collection`

(ANN201)